gtsfm/utils/metrics.py

```diff
@@ -225,7 +225,7 @@ def compute_rotation_angle_metric(wRi_list: List[Optional[Rot3]], gt_wRi_list: L
         A GtsfmMetric for the N rotation angle errors, in degrees.
     """
     errors = []
-    for (wRi, gt_wRi) in zip(wRi_list, gt_wRi_list):
+    for wRi, gt_wRi in zip(wRi_list, gt_wRi_list):
```

Comment:
@akshay-krishnan could you share your python `black` version?

Reply:
Im using the vscode extension: v2023.4.1